src/swapper/swapper.py

```diff
@@ -97,13 +96,21 @@ def copy_and_replace( #: pylint: disable=dangerous-default-value too-many-statem
     def check_table_existence(workspace, table_name):
         with arcpy.EnvManager(workspace=workspace):
             if not arcpy.Exists(table_name):
-                raise Exception(f'{table_name} does not exist in {workspace}')
+                raise FileNotFoundError(f'{table_name} does not exist in {workspace}')
 
     check_table_existence(source_workspace, source_feature_class_name)
```

Comment:
**question**: are we wanting exceptions on empty source to fail the script or should this also be try excepted?

Reply:
I think that we do want empty source to exit swapper with an exception. This shouldn't short circuit forklift as [the pallet does a try/catch](https://github.com/agrc/warehouse/blob/b7e1dee4a19132c4b97713123b1e3c9dc1bb71a5/sgid/SwapperPallet.py#L53).